twake/frontend/src/app/scenes/Apps/Messages/Input/Input.tsx

```diff
@@ -67,18 +69,22 @@ export default (props: Props) => {
         parent_message_id: props.threadId || '',
       },
       props.collectionKey,
-    ).then(message => {
-      setLoading(false);
-      if (
-        messageEditorService.currentEditor ===
-        messageEditorService.getEditorId(props.threadId, props.messageId || '', props.context)
-      ) {
-        autocomplete.focus();
-      }
-      if (!message.parent_message_id) {
-        messageEditorService.openEditor(message.id, props.messageId || '');
-      }
-    });
+    )
+      .then(message => {
+        setLoading(false);
+        if (
+          messageEditorService.currentEditor ===
+          messageEditorService.getEditorId(props.threadId, props.messageId || '', props.context)
+        ) {
+          autocomplete.focus();
+        }
+        if (!message.parent_message_id) {
+          messageEditorService.openEditor(message.id, props.messageId || '');
+        }
+      })
+      .finally(() => {
+        messageEditorService.filesAttachements = {};
```

Comment:
Here you removed all files from all editors (responses, main input etc) I suppose you want to empty only the files attached to the current editor. "messageEditorService.filesAttachements[threadId || "main"] = []"

Reply:
Done :+1: